FlooringOrderingSystem/FlooringOrderingSystem.Data/ProductionOrders.cs

```diff
@@ -0,0 +1,350 @@
+﻿using FlooringOrderingSystem.Models;
+using FlooringOrderingSystem.Models.Interfaces;
+using FlooringOrderingSystem.Models.Responses;
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FlooringOrderingSystem.Data
+{
+    public class ProductionOrders: IOrdersInventory
+    {
+        private readonly string FolderPath = @"C:\Users\Rohit\Desktop\SoftwareGuild\Repos\FlooringOrderingSystem\FlooringOrderingSystem.Data\OrdersList";
+
+        List<Order> orderList = new List<Order>();
+
+
+        public Order LookUpOrder(DateTime OrderDate, int OrderNumber)
+        {
+            Order order = new Order();
+            bool _orderFound = false;
+
+            bool OrderFileExists = CheckOrderFileExist(OrderDate, out string filepath);
+            if (!OrderFileExists)
+            {
+                return null;
+            }
+
+            using (StreamReader reader = new StreamReader(filepath))
+            {
+                string line = reader.ReadLine();
+                while (((line = reader.ReadLine()) != null) && (!_orderFound))
+                {
+                    string[] columns = line.Split(',');
+                    order.OrderNumber = int.Parse(columns[0]);
+                    order.OrderDate = OrderDate;
+                    order.CustomerName = AddCommaintheData(columns[1]);
+                    order.State = columns[2];
+                    order.TaxRate = decimal.Parse(columns[3]);
+                    order.ProductType = columns[4];
+                    order.Area = decimal.Parse(columns[5]);
+                    order.CostPerSquareFoot = decimal.Parse(columns[6]);
+                    order.LaborCostPerSquareFoot = decimal.Parse(columns[7]);
+                    order.MaterialCost = decimal.Parse(columns[8]);
+                    order.LaborCost = decimal.Parse(columns[9]);
+                    order.Tax = decimal.Parse(columns[10]);
+                    order.Total = decimal.Parse(columns[11]);
+
+                    if (order.OrderNumber == OrderNumber)
+                    {
+                        _orderFound = true;
+                    }
+
+                }
+
+            }
+
+
+
+            if (_orderFound)
+            {
+                return order;
+            }
+            else
+            {
+                return null;
+            }
+
+                      
+           
+        }
+
+        public DisplayAllOrdersResponse LoadOrder(DateTime OrderDate)
```

Comment:
instead of returning a response, return an List<Order>

Reply:
Done. Thanks for the advise. I was feeling the pinch of redundancy in the response message from OrderManager and IOrdersInventory but decided to move on to get the project complete.